doc/glossary.rst

```diff
@@ -344,35 +344,12 @@ General Concepts
         likelihoods.
 
     estimator tags
-        A proposed feature (e.g. :issue:`8022`) by which the capabilities of an
-        estimator are described through a set of semantic tags.  This would
-        enable some runtime behaviors based on estimator inspection, but it
-        also allows each estimator to be tested for appropriate invariances
-        while being excepted from other :term:`common tests`.
-
-        Some aspects of estimator tags are currently determined through
-        the :term:`duck typing` of methods like ``predict_proba`` and through
-        some special attributes on estimator objects:
-
-        .. glossary::
-
-            ``_estimator_type``
```

Comment:
These are actually glossary terms and there are links to them elsewhere in the glossary. You'll need to make sure they're still available as glossary entries even if they're no longer (maybe?) considered estimator tags